Resources/Prototypes/_NF/Shipyard/Nfsd/broadhead.yml

```diff
@@ -35,4 +35,4 @@
         - type: StationJobs
           availableJobs:
             Detective: [ 0, 0 ]
-            SecurityOfficer: [ 0, 0 ]
\ No newline at end of file
+            SecurityOfficer: [ 0, 0 ]
```

Comment:
As per new guidelines, NFSD ships should have no available jobs at all (except prisoner where applicable, for now). Change these lines to `availableJobs: {}` :)

Reply:
Done!